coptic_char_generator.py

```diff
@@ -74,34 +74,15 @@ def train_batch(model, optimizer, criterion, data, data_indexes, update=True):
     for i in data_indexes:
         data_item = data[i]
 
-        if data_item.indexes is None:
```

Comment:
don't need this anymore - mask and label will automatically calculate from the original text so we have a fresh start